docs/how_to/single_node.md

```diff
@@ -57,6 +57,10 @@ curl -sS -X POST http://localhost:9003/writetagged -d '{
   "namespace": "default",
   "id": "foo",
   "tags": [
+    {
+      "name": "__name__",
+      "value": "user_sign_up"
```

Comment:
This is a good example, however the best argument for leveraging metrics over logs is for high volume events (rather than less frequent events, which probably don't cost that much to store anyway as logs).

With that in mind, perhaps we should slightly tweak the example.

Maybe something like "user_login"?